functional_test.go

```diff
@@ -36,28 +36,29 @@ import (
 	"github.com/prometheus/common/model"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
-
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials/insecure"
 	json "google.golang.org/protobuf/encoding/protojson"
 )
 
+var allPeers = []guber.PeerInfo{
```

Comment:
No need to expose `allPeers`.  You can use `cluster.GetPeers()`.

Reply:
updated